### ISSUE TITLE
Steps to reproduce SFT-K+ results on cifar10 and cifar100

### ISSUE THREAD

**Issue by nikokks** [SunnySiqi/LNL_K#1]
Hello,

First of all : Congratualtions for your work !

I want to reproduce the cifar10 and cifar100 results with sft-k+, can you show me how to do it ?

Best regards

**Comment by SunnySiqi** [comment]
Hi, thanks for your interests ;) Here are the steps to run the experiments:

**- Build the environment with requirements.txt**
**- Set parameters**
The config files for cifar10/cifar100 with sft+k are at [synthesized_data/hyperparams/multistep/config_cifar10_self_w_rn34.json] (https://github.com/SunnySiqi/LNL_K/blob/main/synthesized_data/hyperparams/multistep/config_cifar10_self_w_rn34.json) and [synthesized_data/hyperparams/multistep/config_cifar100_self_w_rn34.json] (https://github.com/SunnySiqi/LNL_K/blob/main/synthesized_data/hyperparams/multistep/config_cifar100_self_w_rn34.json)
You can set the noise ratio, noise type and other training parameters (learning rate, epochs, etc.) here.
**- Run scripts**
scripts for cifar10/cifar100 with sft+k are at [synthesized_data/scripts/sample_selection_based/self_w.sh](https://github.com/SunnySiqi/LNL_K/blob/main/synthesized_data/scripts/sample_selection_based/self_w.sh)
and [synthesized_data/scripts/sample_selection_based/self_w_cifar100.sh]
(https://github.com/SunnySiqi/LNL_K/blob/main/synthesized_data/scripts/sample_selection_based/self_w_cifar100.sh)
Simply run bash scripts/sample_selection_based/self_w.sh within folder synthesized_data for the experiments.
**- Check the results**
Logs are in the folder saved/log

Hope these could be helpful! Let me know if you have any questions and comments :) 
(Feel free to contact me at wangsiqi49@gmail.com)
